src/main/resources/fabric.mod.json

```diff
@@ -27,9 +27,9 @@
     ],
 
     "depends": {
-        "fabricloader": ">=0.14.9",
+        "fabricloader": ">=0.14.11",
         "fabric-api": "*",
-        "minecraft": "~1.19",
+        "minecraft": ">=1.19.2",
```

Comment:
`">="` should be `"~"`, oops :v

(`">=1.19.2"` includes *every* version above or equal to 1.19.2 (incl. 1.20), while `"~1.19.2"` only includes 1.19's *minor versions* above or equal to 1.19.2 (i.e. 1.19.2, 1.19.3, etc., but NOT 1.20 or above).

Reply:
Also, leaving it as `"~1.19"` probably would've been fine...